packages/web/components/notifi/notifi-context.tsx

```diff
@@ -0,0 +1,93 @@
+import { WalletAccount } from "@cosmos-kit/core";
+import { NotifiContext } from "@notifi-network/notifi-react-card";
+import { observer } from "mobx-react-lite";
+import React, {
+  FunctionComponent,
+  PropsWithChildren,
+  useEffect,
+  useState,
+} from "react";
+
+import { useStore } from "../../stores/";
+import { NotifiConfigContext } from "./notifi-config-context";
+import { NotifiModalContextProvider } from "./notifi-modal-context";
+
+interface RequiredInfo {
+  account: WalletAccount;
+  chainId: string;
+}
+
+const NotifiContextProviderImpl: FunctionComponent<PropsWithChildren<{}>> = ({
```

Comment:
IDK if this has caused you problems, but when you make references to MobX observable objects from our root store (via useStore), you need to simply wrap the component function definition in `observer` so MobX knows when to rerender.

Reply:
Np, done it ty.